webapp/graphite/future.py

```diff
@@ -0,0 +1,55 @@
+"""Lightweight Future implementation.
+
+Before doing that we should evalluate the use of concurrent.futures
+first, which could also replace FetchInProgress.
+"""
+import abc
+
+from graphite.logger import log
+
+
+class Future(object) :
+    def __init__(self, *args, **kwargs) :
+        super(Future, self).__init__(*args, **kwargs)
```

Comment:
I'm not sure if i just don't get what's happening here, but doesn't that just always call `__init__()` on the `object` it's inheriting from? what does that do?